lib/wallet/widgets/dialogs/account_dialog.dart

```diff
@@ -210,7 +212,8 @@ class AccountDialog extends StatelessWidget {
                               ),
                               Text(
                                 'Show on Polygonscan',
-                                style: textStyle(Colors.grey[400]!, 15, false),
+                                style: textStyle(Colors.grey[400]!, 17,
+                                    false), //change font size
```

Comment:
Same thing here. Either use` FittedBox` or use the `AutoSizeText` to make the text size dynamic. I would start with using `AutoSizeText` and then go from there. Also please remove the comments.